app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -105,7 +111,7 @@ public void onFailure(Call<List<com.uwetrottmann.trakt5.entities.Show>> call, Th
     }
 
     // get it by genre from recommended shows call
-    public void fetchRecommendedShows(Shows traktShows, ShowAdapter adapter, User user) {
+    public void fetchRecommendedShows(Shows traktShows, ShowAdapter adapter) {
         // from trakt wrapper -> call to get related shows and send to adapter
         try {
             // enqueue to do asynchronous call and execute to do it synchronously
```

Comment:
This if else below is getting pretty long.  I would pull it out into two separate methods to improve readability